about/exploring.md

```diff
@@ -18,16 +20,19 @@ The fact that over the past few years, _billions_ of new devices have come onlin
 ## Why embedded browsers are different
 For many of us, a browser is an application like the one you're probably using now.  You click an icon on your graphical operating system, navigate somewhere with a URL bar, search, and so on.  You have bookmarks and tabs that you can drag around, and lots of other features.
 
-For most embedded systems, this isn't the case at all.  For most embedded systems there isn't a common desktop OS&nbsp;— in fact, _you_ create the OS.  Part of the reason for this is that, by and large, they are not general-purpose computing devices.  They have slightly different security concerns, and they also run on often radically different sorts of hardware than your desktop, laptop or many mobile devices.
+In contrast, an embedded browser is contained within another application or is built for a specific purpose and runs in an embedded system, and the application controlling the embedded browser does not provide all the typical features of browsers running in desktops.
+
+Furthermore, for most embedded systems there isn't a common desktop OS for the browser to run on&nbsp;— in fact, the OS is built specifically for that type of device, and the embedded browser is built along with it.  Part of the reason for this is that, by and large, they are not general-purpose computing devices.  They have slightly different security concerns, and they also run on often radically different sorts of hardware than desktop, laptop or many mobile devices.
```

Comment:
Also where we use the acronym we could make use of the `<abbr>` tag:

> [...] the \<abbr title="Operating System">OS\</abbr> is built specifically [...]

Reply:
done